target/start-mailserver.sh

```diff
@@ -637,6 +637,9 @@ function _setup_dovecot_quota() {
     if [ "$ENABLE_LDAP" = 1 ]; then
       notify 'inf' "Dovecot quota is not implemented with LDAP."
 
+      # LDAP => postfix service for dovecot quota not enabled
```

Comment:
It looks like you closed the if statement wrong? Before the SMTP_ONLY check was outside the ENABLE_LDAP check, now it is inside?

Reply:
The last commit remove the nested condition, it's cleaner.
The function should only disable or enable quota.

Quota is disabled when LDAP is enabled or when SMTP_ONLY is enabled, otherwise dovecot quota is enabled.

Do you agree this logic?